site-src/blog/2023/0823-welcome-gamma.md

```diff
@@ -0,0 +1,160 @@
+</small>
+
+We are thrilled to announce the v0.8.0 release of the Gateway API! With this
+release, the work the GAMMA initiative has been doing over the past year has
+reached [Experimental status][status], and the Gateway and HTTPRoute resources
+gain some new capabilities. We look forward to your feedback!
+
+## What is the GAMMA initiative?
+
+Almost from the point where the Gateway API was itself implementable,
+questions have been raised about how it could be used for configuring service
+meshes. The [GAMMA initiative][gamma], started in 2022, is a dedicated
+vendor-neutral workstream within the Gateway API, working on examining how
+best to fit the service mesh into the framework of the Gateway API resources,
+without requiring users of the Gateway API to relearn everything they
+understand about the Gateway API itself.
+
+Over the last year, GAMMA has dug deeply into the challenges and possible
+solutions around using the Gateway API for service mesh. The end result is a
+small number of [enhancement
+proposals](https://gateway-api.sigs.k8s.io/v1beta1/contributing/gep/) that
+subsume many hours of thought and debate, and provide a minimum viable path to
+allow the Gateway API to be used for service mesh. Of particular note:
+
+- [GEP-1324](https://gateway-api.sigs.k8s.io/geps/gep-1324/) provides an
+  overview of the GAMMA goals and some important definitions. This GEP is well
+  worth a read for its discussion of the problem space.
+- [GEP-1426](https://gateway-api.sigs.k8s.io/geps/gep-1426/) defines how to
+  use Gateway API route resources, such as HTTPRoute, to manage traffic within
+  a service mesh.
+- [GEP-1686](https://gateway-api.sigs.k8s.io/geps/gep-1686/) builds on the
+  work of [GEP-1709](https://gateway-api.sigs.k8s.io/geps/gep-1709/) to define
+  a _conformance profile_ for service meshes to be declared conformant with
+  the Gateway API.
+
+Although these are [Experimental][status] APIs, note that they are available
+in the [`standard` release channel][ch], since the GAMMA initiative has not
+needed to introduce new resources or fields to date.
+
+### How will mesh routing work when using the Gateway API?
+
+You can find all the details in the [Gateway API Mesh routing
+documentation][mesh-routing] and [GEP-1426], but the short version for Gateway
+API v0.8.0 is that an HTTPRoute can now have a `parentRef` that is a Service,
+rather than just a Gateway. We anticipate future GEPs in this area as we gain
+more experience with service mesh use cases -- binding to a Service makes it
+possible to use the Gateway API with a service mesh, but there are several
+interesting use cases that remain difficult to cover.
+
+As an example, you might use an HTTPRoute to do an A-B test in the mesh as
+follows:
+
+```yaml
+  apiVersion: gateway.networking.k8s.io/v1beta1
+  kind: HTTPRoute
+  metadata:
+    name: bar-route
+  spec:
+    parentRefs:
+      - name: demo-app
+          port: 5000
+          kind: Service
```

Comment:
Nit: Indentation is off here: https://deploy-preview-2181--kubernetes-sigs-gateway-api.netlify.app/blog/2023/0823-welcome-gamma/

Reply:
Also missing `group: ""`

Reply:
Nice catch, many thanks!